src/main/java/com/moodybluez/enterprise/dao/EntrySQLDAO.java

```diff
@@ -14,26 +17,28 @@
     @Autowired
     private EntryRepository entryRepository;
 
+    Logger LOG = LoggerFactory.getLogger(this.getClass());
+
     @Override
     public Entry save(Entry entry) {
         return entryRepository.save(entry);
     }
 
     @Override
     public List<Entry> fetchByMonth(int year, int month) {
-        List<Entry> t= entryRepository.findByMonth(year,month);
-        return t;
+        return entryRepository.findByMonth(year,month);
```

Comment:
No need to define variable and return, just return what the method returns if you don't need to modify it.